web-interface/api/logic.py

```diff
@@ -101,13 +101,33 @@ def AppendDiskInfo(diskMountPoint):
                  })
     pass
 
-# Going to be updating to increase efficiency of nmap scans.
+
+# Create a list of ports and services, and create a class to define ports and services.
+ServiceList = []
+class PortInfo:
+    def __init__(self, port, service, IsActive):
+        self.port = port
+        self.service = service
+        self.IsActive = IsActive
+# Define ports. IsActive will be "False" by default. Becomes "True" it the port is Open.
+port22 = PortInfo(22, 'ssh', False)
+port80 = PortInfo(80, 'rpms/web', False)
+port139 = PortInfo(139, 'samba/netbios', False)
+port445 = PortInfo(445, 'samba/tcp', False)
+port5000 = PortInfo(5000, 'rpms/api', False)
+port8384 = PortInfo(8384, 'syncthing/web', False)
+port9002 = PortInfo(9002, 'lms/web', False)
+port9090 = PortInfo(9090, 'lms/telnet', False)
+port9091 = PortInfo(9091, 'transmission/web', False)
+
+# Save the list of ports as a string "PortList" for passing to Nmap.
+portList = f"{port22.port},{port80.port},{port139.port},{port445.port},{port5000.port},{port8384.port},{port9002.port},{port9090.port},{port9091.port}"
```

Comment:
This should go dynamically, by iterating over a list (as described in a previous) comment.